### DISCUSSION TITLE
Preview of Rune 0.0.0-dev.6

### DISCUSSION THREAD

**Discussion by Losses** [Losses/rune#6] [General]
# Features
- Design icons in Fluent, Gnome, Breeze, and Papirus styles
- Provide a complete Flake build and development environment

# Bug Fixes
- Add responsive design to the control bar (narrow screens)
- Add responsive design to the Cover Art Wall (narrow screens)
- Fix the issue with the Cover Art Wall Page not having a light mode
- Fix the issue where scan tasks at the end of the queue cannot be completed

https://github.com/user-attachments/assets/7c56e9ac-1e20-405b-b281-dce471870b66

**Comment by bxb100** [comment]
telegram 有群吗
